tests/unit/number.js

```diff
@@ -410,14 +427,14 @@ define([
 
 
 			// NumberRegression.Test4052223
-			checkParse({ pattern: '#,#00.00' }, 'abc3');
+			checkParse({ pattern: '#,#00.00' }, 'abc3', NaN);
```

Comment:
Due to modification in the `checkParse()` function, the expected value is now required. Since `abc3` is not a number, the expected value is `NaN`.